notebooks/query_uploaded_database.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">data = yaml_file # dataset.yaml path
imgsz= 640&nbsp; # inference size (height, width)
conf_thres = 0.05&nbsp; # confidence threshold
iou_thres = 0.05&nbsp; # NMS IOU threshold
max_det = 1000&nbsp; # maximum detections per image
project = 'runs/detect'&nbsp; # save results to project/name
name ='exp'&nbsp; # save results to project/name
</pre><blockquote>Don't be afraid to make a name long. A long descriptive name is better than a short enigmatic name. A long descriptive name is better than a long descriptive comment.</blockquote><blockquote><strong>Robert C. Martin (Clean Code)</strong></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>